src/graphdb.js

```diff
@@ -193,34 +256,65 @@ function graphDBInferenceSchema (graphbSchema, addMutations) {
-            let nodeCase = toPascalCase(node.label);         
-            r += `\tcreateNode${nodeCase}(input: ${nodeCase}Input!): ${nodeCase}\n`;
-            r += `\tupdateNode${nodeCase}(input: ${nodeCase}Input!): ${nodeCase}\n`;
-            r += `\tdeleteNode${nodeCase}(_id: ID!): Boolean\n`;
+            let nodeCase = replaceCleanseLabel(node.label);
+            let invalidNode = node.label !== nodeCase ? true : false;
+            nodeCase = toPascalCase(nodeCase);
+            if (invalidNode) {
+                r += `\tcreateNode${nodeCase}(input: ${nodeCase}Input!): ${nodeCase} @alias(property: "createNode${node.label}")\n`;
+                r += `\tupdateNode${nodeCase}(input: ${nodeCase}Input!): ${nodeCase} @alias(property: "updateNode${node.label}")\n`;
+                r += `\tdeleteNode${nodeCase}(_id: ID!): Boolean @alias(property: "deleteNode${node.label}")\n`;
+            }
+            else {
+                r += `\tcreateNode${nodeCase}(input: ${nodeCase}Input!): ${nodeCase}\n`;
+                r += `\tupdateNode${nodeCase}(input: ${nodeCase}Input!): ${nodeCase}\n`;
+                r += `\tdeleteNode${nodeCase}(_id: ID!): Boolean\n`;
+            }
         });    
 
         gdbs.edgeStructures.forEach(edge => {
             edge.directions.forEach(direction => {
-            let edgeCase = toPascalCase(edge.label);
-            let fromCase = toPascalCase(direction.from);
-            let toCase = toPascalCase(direction.to);    
-            if (edge.properties.length > 0) {                                            
-                r += `\tconnectNode${fromCase}ToNode${toCase}Edge${edgeCase}(from_id: ID!, to_id: ID!, edge: ${edgeCase}Input!): ${edgeCase}\n`;
-                r += `\tupdateEdge${edgeCase}From${fromCase}To${toCase}(from_id: ID!, to_id: ID!, edge: ${edgeCase}Input!): ${edgeCase}\n`;
-            } else {
-                r += `\tconnectNode${fromCase}ToNode${toCase}Edge${edgeCase}(from_id: ID!, to_id: ID!): ${edgeCase}\n`;            
-            }
-            r += `\tdeleteEdge${edgeCase}From${fromCase}To${toCase}(from_id: ID!, to_id: ID!): Boolean\n`;
+                let fromCase = replaceCleanseLabel(direction.from);
+                let toCase = replaceCleanseLabel(direction.to);
+                let edgeCase = replaceCleanseLabel(edge.label);
+                let invalidDir = direction.from !== fromCase || direction.to !== toCase || edge.label !== edgeCase ? true : false;
+                fromCase = toPascalCase(fromCase);
+                toCase = toPascalCase(toCase);
+                edgeCase = toPascalCase(edgeCase);
+
+                if (edge.properties.length > 0) {               
+                    if (invalidDir) {
+                        r += `\tconnectNode${fromCase}ToNode${toCase}Edge${edgeCase}(from_id: ID!, to_id: ID!, edge: ${edgeCase}Input!): ${edgeCase} @alias(property: "connectNode${direction.from}ToNode${direction.to}Edge${edge.label}")\n`;
```

Comment:
I don't think these aliases are used for edge mutations. The APIs seem to work without it (I tried commenting out this alias logic and then connecting an ec2 node to a vpc node and it still works).